cockatrice/src/chatview.cpp

```diff
@@ -326,7 +326,12 @@ void ChatView::mousePressEvent(QMouseEvent *event)
                     break;
                                       }
                 case Qt::LeftButton :{
-                    emit addMentionTag("@" + userName);
+                    if (event->modifiers() == Qt::ControlModifier) {
+                        const int delimiterIndex = hoveredContent.indexOf("_");
+                        const QString userName = hoveredContent.mid(delimiterIndex + 1);
```

Comment:
Are you redeclaring these in this scope? I see them defined on line 320-21, are they not visible at this location?